src/Stratis.Bitcoin/Connection/PeerBanningBehavior.cs

```diff
@@ -49,13 +49,13 @@ protected override void AttachCore()
         {
             this.logger.LogTrace("()");
 
-            var node = this.AttachedPeer;
-            if (node.State == NetworkPeerState.Connected)
+            var peer = this.AttachedPeer;
```

Comment:
unvar

Reply:
For some reason Github wont show all files changed and would mark this comment as a resolved one. The changes for PeerBanningBehavior are still there but this file is not included in the [github changed files view](https://github.com/stratisproject/StratisBitcoinFullNode/pull/1055/files) 
Odd.

Reply:
if it is not on the list, it means this file haven't been changed in this PR, it could be in merge of other PR while syncing, but it is unrelated to this PR

therefore offtopic and won't fix, although you are right of course